--- src/main.rs
+++ src/main.rs
@@ -78,6 +78,9 @@
         Some("daemon") => {
             // Just like we did with --force in init, we do same with daemon options for --status
             if has_flag(&args, "--status") {
                 // This function will literally just tell us if the daemon is actively working or not then close the program(return).
                 return cli::status::run().map_err(Into::into);
             }
+            if has_flag(&args, "--flush-now") {
+                return cli::status::force_flush().map_err(Into::into);
+            }
             // Without the flag, it sets up the daemon.
             info!("Starting bruh daemon…");
             daemon::run().await?;
